tools/reflection_generator/java_method.py

```diff
@@ -731,20 +773,20 @@ def GenerateWrapperInterface(self):
     return template.substitute(value)
 
   def GenerateMethodsStringForBridge(self):
-    self.PrepareStrings()
     if self._is_constructor:
       return self.GenerateBridgeConstructor()
     elif self._is_static:
       return self.GenerateBridgeStaticMethod()
     else:
-      return '%s%s%s%s' % (
-          '    private Method %s;\n' % self._method_declare_name,
+      return '%s\n%s\n%s\n%s\n' % (
           self.GenerateBridgeOverrideMethod(),
           self.GenerateBridgeWrapperMethod(),
-          self.GenerateBridgeSuperMethod())
+          self.GenerateBridgeSuperMethod(),
+          '    private ReflectMethod %s = \
+new ReflectMethod(null, null, "%s");\n' %
+              (self._method_declare_name, self._method_name))
```

Comment:
The method call may be reserved if the reflection layer is not available yet. In this case, it's helpful that we can see the name of reserved method in log. So the method object is initialized with the method name but not just a null method